src/js/apps/admin/program/workflows/workflows_app.js

```diff
@@ -20,6 +20,16 @@ export default App.extend({
     this.actions = actions;
 
     this.showChildView('content', new ListView({ collection: actions }));
+
+    if (_DEVELOP_) {
+      this.showChildView('add', new AddActionDroplist({
+        picklistEvents: {
+          'picklist:item:select': ({ model }) => {
+            model.attributes.onSelect(this.program);
```

Comment:
we should use `model.get('onSelect')(this.program)`  While it's perfectly ok here, backbone cautions against using `attributes` in general, so we've stuck with that stylistically. 
I do think this event would be better on the view like:
```js
  picklistEvents: {
    'picklist:item:select': 'onSelect',
  },
  onSelect({ model }) {
    model.get('onSelect')();
  },
```

And I think the collection and `onSelect => () {`s using `this.program.id` can be here in the app.